### ISSUE TITLE
Crashes on linux system

### ISSUE THREAD

**Issue by playoff4157** [KRTirtho/spotube#154]
**Describe the bug**
Crashes as soon as it opens. Sometimes works for a few seconds but then crashes.
Shows this kind of message when I open from terminal: 

> nouveau: 	0x00000000
nouveau: 	0x00000000
nouveau: 	0x00000000
nouveau: 	0x00000000
nouveau: 	0x200406c0
nouveau: 	0x00000000
nouveau: 	0x00014000
nouveau: 	0x0000006e
nouveau: 	0x1000f010
spotube: ../nouveau/pushbuf.c:728: nouveau_pushbuf_data: Assertion `kref' failed.

and sometimes this:
>AudioPlayers: Could not query current position.

**To Reproduce**
1. Install using flatpak
2. Open the app
3. Crashes

**Expected behavior**
Should Open and play songs

**Installation Source/Method**
- Package Manager: flatpak, App version : 2.3.0

**Desktop (please complete the following information):**
 - OS: manjaro linux, DE: gnome 42.3, using nvidia opensource drivers

**Comment by KRTirtho** [comment]
Try the AUR version `spotube-bin`. Probably something packaged inside the Flatpak version is causing this driver (.so) mismatch

**Comment by KRTirtho** [comment]
@playoff4157 did your issue resolve?

**Comment by playoff4157** [comment]
Nope. This is an issue with flutter's working with noveau open source drivers. Similar [issue](https://github.com/harmonoid/harmonoid/issues/304) with harmonoid.

**Comment by aopom** [comment]
I have the same issue with flatpak on NixOS and nouveau open source nvidia drivers :thinking: 
```
nouveau: kernel rejected pushbuf: No such file or directory
nouveau: ch9: krec 0 pushes 0 bufs 2 relocs 0
nouveau: ch9: buf 00000000 00000003 00000004 00000004 00000000
nouveau: ch9: buf 00000001 00000006 00000004 00000000 00000004
```

**Comment by aopom** [comment]
From the linked harmonoid issue, I discovered that I can run it with
```bash
LIBGL_ALWAYS_SOFTWARE=1 flatpak run com.github.KRTirtho.Spotube
```
But with software-rendered graphics, the CPU load is huge.
